reference/pipeline/stages/index.md

```diff
@@ -89,15 +89,23 @@ Server Groups and any existing Server Groups. Enabling a Server Group also
 re-enables auto-scaling policies, if applicable.
 
 ### Find Artifact From Execution
-Find and bind an artifact from another pipeline execution.
+Find and bind an artifact from a different execution to the current one,
+including artifacts from different Spinnaker applications. To do this, specify
+the type of artifact (GCS, GitLab, Docker, etc) and its path. You can specify
+that Spinnaker should only consider executions that meet certain criteria, such
+as ones that have completed successfully or that are currently running.
+Spinnaker will always return the artifact from the most recent execution that
+matches your criteria.
```

Comment:
Lars pointed out that it should actually be the image name -- turns out "path" was GCS-specific :) I think that becomes clear enough that I think the example isn't necessary, particularly since the hover text for each type of image is pretty informative.